core/src/main/java/org/elasticsearch/action/get/MultiGetRequest.java

```diff
@@ -371,9 +403,9 @@ public MultiGetRequest add(@Nullable String defaultIndex, @Nullable String defau
 
     public static void parseDocuments(XContentParser parser, List<Item> items, @Nullable String defaultIndex, @Nullable String defaultType, @Nullable String[] defaultFields, @Nullable FetchSourceContext defaultFetchSource, @Nullable String defaultRouting, boolean allowExplicitIndex) throws IOException {
```

Comment:
Can this be private? Seems only be used from "add" and the other parseDocuments() method.